### ISSUE TITLE
Changelog command not replacing %s

### ISSUE THREAD

**Issue by carlosl** [webpro/release-it#150]
I currently maintain a project that has a manually updated changelog. Contributors manually add content under the '[Unreleased]' section at the top of the Changelog.log file. We follow this -
 http://keepachangelog.com/en/1.0.0/ - pattern. According to the release-it docs, any %s would be replaced by the version number.

So I have a sed command that updates this header with the release number and date:
```
sed -e "s/\[Unreleased\]/\[%s\]\ \-\ `date +%Y-%m-%d`/" CHANGELOG.md
```
It works as expected except that  for replacing the %s (which I expect release-it to replace later on.

So in my release-it.json I added the following, escaping the chars I needed, (the -i .bak just replaced the file in place and creates a backup file which I gitignore) etc...

```
{
"changelogCommand": "sed -e \"s/\\[Unreleased\\]/\\[%s\\]\\ \\-\\ `date +%Y-%m-%d`/\" CHANGELOG.md -i .bak"
}
```
It works the same as before though without replacing the %s characters. Does it not actually replace it in all instances?

Thanks in advance for your help.

**Comment by webpro** [comment]
First of all, I agree the docs aren't very clear in this regard. A few notes:

* The `%s` is replaced with the version number in `src.commitMessage`, `src.tagName`, `src.tagAnnotation`, `github.releaseName`.
* The `%s` in the `changelogCommand` is part of the git log printf-like formatting, where `%s` is replaced with the commit "subject" (see [Git - pretty-formats Documentation](https://git-scm.com/docs/pretty-formats)).
* In the build commands (such as `buildCommand`), the config itself can be used to replace things in the command. Also see [command-hooks](https://github.com/webpro/release-it#-command-hooks).

The short answer would be: no, nothing is replaced by release-it in the `changelogCommand`. 

This might be a nice, non-breaking addition, though. You up for a PR?

**Comment by webpro** [comment]
I've just released [v4.4.0](https://github.com/webpro/release-it/releases/tag/4.4.0) which supports replacements in the `changelogCommand`. Note that `%s` should be `${version}` in your example.
